app/views/car_models/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>New Car Model</h1>
+
```

Comment:
<% @car_model.errors.each do |attribute, message| %>
  <p>The <%= attribute %> <%= message %></p>
<% end %>